src/server/system_services/schemas/replication_configuration_schema.js

```diff
@@ -0,0 +1,36 @@
+/* Copyright (C) 2016 NooBaa */
+'use strict';
+
+module.exports = {
+    id: 'replication_configuration_schema',
+    type: 'object',
+    required: [
+        '_id',
+        'replication_policy',
+    ],
+    properties: {
+
+        // identifiers
+        _id: { objectid: true },
+        replication_policy: {
```

Comment:
maybe rename it to `rules`, so in your code you will have `replication_policy.rules` instead of `replication_policy.replications_policy`

Reply:
renamed